ibis/backends/bigquery/converter.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import annotations
+
+from ibis.formats.pandas import PandasData
+
+
+class BigQueryPandasData(PandasData):
+    @classmethod
+    def convert_GeoSpatial(cls, s, dtype, pandas_type):
+        import geopandas as gpd
+        import shapely as shp
+
+        return gpd.GeoSeries(shp.from_wkt(s))
```

Comment:
I'm deliberately not converting to binary for now. If it's a performance issue at some point we can add the transform in an override of `ibis.backends.bigquery.Backend._to_sqlglot`.